packages/fxa-admin-panel/src/index.tsx

```diff
@@ -28,12 +29,29 @@ const client = new ApolloClient({
   cache: new InMemoryCache(),
 });
 
-ReactDOM.render(
-  <ApolloProvider {...{ client }}>
-    <App {...{ config }} />
-  </ApolloProvider>,
-  document.getElementById('root')
-);
+try {
+  readConfigFromMeta(headQuerySelector);
+
+  sentryMetrics.configure({
+    release: config.version,
```

Comment:
@dschom I noticed we use `config.sentry.version` in at least Settings, which I think should probably reference `config.version`. I think we also reference the `package.json` file like `const release = require('../package.json').version;` in a few places instead of packages' `version.ts` files, and I think we might need to do that and possibly modify the version file to check for the release number differently before #14050 can make it.